ios/Framework/Core/Auth/LoginScreenlet/Interactors/LoginCookieInteractor.swift

```diff
@@ -12,26 +12,22 @@
 * details.
 */
 import UIKit
-import LRMobileSDK
 
-open class LoginCookieInteractor: Interactor, LRCallback {
+open class LoginCookieInteractor: ServerConnectorInteractor {
 
 	open let emailAddress: String
```

Comment:
This doesn't necessarily has to be a emailAddress, it could be an screen name or an userId, Do you think we should add the change to this PR? A name like 'username' would fit better. WDYT?

Reply:
Yes, I agree. It could be `emailAddress`, `userId` or `screenName`, so maybe it's more understandable to change the name to `username` 👍 I'll change it, too.